### ISSUE TITLE
Imperative mood for default messages

### ISSUE THREAD

**Issue by gitfineon** [remkop/picocli#791]
For many tools the title and description of commands and options are in imperative mood.
How about changing the default messages like "<del>Displays</del> Display help information about ..."

**Comment by remkop** [comment]
Thanks for looking into this. I believe picocli already uses imperative language; the standard help options look like this:

```
Usage: some-command
  -h, --help               Show this help message and exit.
  -V, --version            Print version information and exit.
```

The above two options are added when you define a command with [mixinStandardHelpOptions](https://picocli.info/#_mixin_standard_help_options):
```
@Command(mixinStandardHelpOptions = true) class SomeCommand {}
```

Am I missing something?

**Comment by gitfineon** [comment]
You are right, didn't found any other then #796.
This showed up on including the HelpCommand.class as subcommand.

**Comment by remkop** [comment]
PR merged. Thank you for the contribution!

**Comment by remkop** [comment]
I reverted this commit for now:
it broke some of the tests that assert the previous HelpCommand header.
This is to be expected, but I want to avoid making breaking changes in patch releases like 4.0.3.

I will re-apply this change in a later release (likely 4.1).

**Comment by gitfineon** [comment]
Of course, np, ... was able to overwrite it with the i18n properties file

**Comment by remkop** [comment]
Thank you for your patience.

This has been fixed in https://github.com/remkop/picocli/pull/1579 and https://github.com/remkop/picocli/pull/1578
and will be included in the picocli 4.7.0 release.
